### PR TITLE
Option set incorrectly for building tests

### PR DESCRIPTION
**Pull request by paulej**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -2,9 +2,9 @@ cmake_minimum_required(VERSION 3.13)
 
 # Build tests by default only if not a sub-project
 if(DEFINED PROJECT_NAME)
-    option(QUICR_BUILD_TESTS "Build tests for quicr" ON)
-else()
     option(QUICR_BUILD_TESTS "Build tests for quicr" OFF)
+else()
+    option(QUICR_BUILD_TESTS "Build tests for quicr" ON)
```
Comment (GhostofCookie) [inline CMakeLists.txt:7]: Is this correct? PROJECT_NAME is only defined if this is a subproject?

Comment (GhostofCookie) [reply CMakeLists.txt:7]: Are the ON/OFF backwards?

Comment (paulej) [reply CMakeLists.txt:7]: If the project name is defined, it's a subproject (i.e., it's included inside of a different project). In that case, we want tests turned off.

Comment (GhostofCookie) [reply CMakeLists.txt:7]: Ah, okay. Makes sense. Thanks!

**Review by GhostofCookie — APPROVED** [review]

**Pull request merged**